src/main/kotlin/frc/robot/subsystems/hood/Hood.kt

```diff
@@ -73,9 +83,28 @@ class Hood private constructor(private val io: HoodIO) : SubsystemBase() {
     fun setRestingAngle(): Command = setAngle(HoodConstants.RESTING_ANGLE).withName("Set Resting Angle Hood")
 
     @AutoLogOutput(key = "Hood/Pose")
-    private fun getPose3d() : Pose3d = Pose3d(HoodConstants.ROOT_POSITION, Rotation3d(0.0, getAngle().plus(HoodConstants.SIMULATION_OFFSET).`in`(Units.Radians), 0.0))
+    private fun getPose3d(): Pose3d = Pose3d(
+        HoodConstants.ROOT_POSITION,
+        Rotation3d(0.0, getAngle().plus(HoodConstants.SIMULATION_OFFSET).`in`(Units.Radians), 0.0)
```

Comment:
`+`? Instead of `plus()`
I'd create an extension function if it doesn't work

Reply:
I don't see a problem with it being a method and not the operator, it doesn't really diminish the readability.